tests/test_discount.py

```diff
@@ -0,0 +1,31 @@
+from pywr.parameters import Parameter, load_parameter
+import datetime
+
+class DiscountFactorParameter(Parameter):
```

Comment:
This parameter looks good, but it should be placed in `pywr/parameters/parameters.py` so its included in the library. Placed here it's in a test module. Instead this module should contain a test function that loads the `discount.json` model you added, and then test that the parameter loads correctly. Another test could test that the outputs of the parameter are the expected values. See `tests/test_parameters.py` for how this is done for some other parameters.

Reply:
Thanks James, I have moved the discount parameter to `pywr/parameters/parameters.py` and created a test in `tests/test_parameters.py` to load the `discount.json` model. I have not added yet a test that checks that the outputs of the parameter are the expected values. Should I create a list of expected values for the discount factor in each year based on a 3.5% discount factor and check that the outputs have the same values?